packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -272,6 +289,12 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public rowClassNameFn?: (row: object, index: number) => string;
 
+	/**
+	 * Function to see if a row is collapsed
+	 */
+	@Input()
```

Comment:
I would keep it because it's used inside Stark Table to check if a row is expanded and `this.expandedRow === row` will not always work.

Reply:
You compare an object directly to another by default now so this might not always work if your row/object gets mutated along the way.

Reply:
I have the feeling that this should not happen since the row comes from the dataset of the table...

Nevertheless, if you think it's better to keep it, we can it as this for now 😊